bofire/data_models/domain/features.py

```diff
@@ -537,6 +538,10 @@ def get_bounds(
             experiments (Optional[pd.DataFrame], optional): Dataframe with input features.
                 If provided the real feature bounds are returned based on both the opt.
                 feature bounds and the extreme points in the dataframe. Defaults to None,
+            reference_experiment (Optional[pd.Serues], optional): If a reference experiment provided,
```

Comment:
Because it is already in the signature of the function, or what do you mean? Problem is somehow that we have this in most of our docstrings. How to proceed then there in general?

Reply:
Exactly. We already have it as type hint. I would not repeat it in the docstring. I would never repeat it in the docstring. We do not have to change this for everything right now. I hope that the doc-extraction-tool makes use of the type hints.